### PR TITLE
Add syntax highlighting

### LINKED ISSUE #1: Add syntax highlighting

### PR DESCRIPTION
**Pull request by Evangelink**
Fix #1

### CODE REVIEW

File: sonar-css-plugin/src/main/java/org/sonar/css/plugin/MetricSensor.java [hunk 1]
```diff
@@ -59,9 +57,45 @@ public void execute(SensorContext context) {
...
+
+          case STRING:
+            highlightingType = TypeOfText.STRING;
+            break;
+
+          case WORD:
+            if (Character.isDigit(currentToken.text.charAt(0)) || currentToken.text.matches("^#[0-9a-fA-F]+$")) {
+              highlightingType = TypeOfText.CONSTANT;
+            } else if (nextToken != null && nextToken.text.equals(":")) {
+              highlightingType = TypeOfText.KEYWORD_LIGHT;
```
Comment (vilchik-elena) [inline sonar-css-plugin/src/main/java/org/sonar/css/plugin/MetricSensor.java]: @Evangelink may be we could use other type to make it more obvious?

File: sonar-css-plugin/src/main/java/org/sonar/css/plugin/MetricSensor.java [hunk 2]
```diff
@@ -59,9 +57,45 @@ public void execute(SensorContext context) {
...
+            } else if (currentToken.text.startsWith(".") || (nextToken != null && nextToken.text.startsWith("{"))) {
```
Comment (vilchik-elena) [inline sonar-css-plugin/src/main/java/org/sonar/css/plugin/MetricSensor.java]: @Evangelink only one token?

File: sonar-css-plugin/src/main/java/org/sonar/css/plugin/MetricSensor.java [hunk 3]
```diff
@@ -59,9 +57,45 @@ public void execute(SensorContext context) {
...
+              highlightingType = TypeOfText.KEYWORD;
+            }
+            break;
+
+          case AT_WORD:
+            highlightingType = TypeOfText.ANNOTATION;
+            break;
+
+          default:
```
Comment (vilchik-elena) [inline sonar-css-plugin/src/main/java/org/sonar/css/plugin/MetricSensor.java]: @Evangelink you don't need default

**Comment by Evangelink** [comment]
Merging as is and I will improve it with the new tokenizer.

**Pull request merged**